From 60719b49aa514b90d1ff817ee6a8129fae83fb24 Mon Sep 17 00:00:00 2001
From: Vladimir Ermakov <vooon341@gmail.com>
Date: Thu, 24 Jul 2025 13:30:41 +0200
Subject: [PATCH 3/3] cargo: use system libzstd

Signed-off-by: Vladimir Ermakov <vooon341@gmail.com>
---
 Cargo.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Cargo.toml b/Cargo.toml
index 6d216af1..6c93f193 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -168,7 +168,7 @@ udev = "0.9"
 url = "2.1"
 walkdir = "2"
 xdg = "2.2"
-zstd = "0.13"
+zstd = { version = "0.13", features = [ "bindgen", "pkg-config" ] }
 zstd-safe = "7"
 
 [dependencies]
-- 
2.52.0

